doc/news/nightly_build.rst

```diff
@@ -0,0 +1,3 @@
+**Changed:**
+
+* Changed download instructions for "nightly build" installer.
```

Comment:
```suggestion
**Fixed:**

* Fixed download instructions for "nightly build" installer.
```

The idea of "Changed" is that something might need changing in dependent code. I think "fixed" is better therefore.